benchmark/storage/metric_benchmark.cpp

```diff
@@ -0,0 +1,165 @@
+#include <memory>
+#include <string>
+#include <thread>  //NOLINT
```

Comment:
Don't include `thread`. It is included implicitly. Check `logging_benchmark.cpp`

Reply:
Changed